recipes/xege/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class XegeConan(ConanFile):
+    name = "xege"
```

Comment:
Seems this is a fork

https://github.com/wysaid/xege

```suggestion
    name = "wysaid-xege"
    provides = "xege"
```

Reply:
Seems weird the "upstream" has no code

Reply:
Yes, the original repo of this library doesn't update anymore. And `wysaid` improved it a lot, like building with CMake, providing offical website, etc.

So It's necessary to change the package name?

Reply:
No. Well see what others think but I fine for me

Reply:
I think it is fine too, the fork really looks like the official EGE